frontend/workflows/serverexperimentation/src/index.tsx

```diff
@@ -9,7 +9,7 @@ const register = (): WorkflowConfiguration => {
       contactUrl: "mailto:hello@clutch.sh",
     },
     path: "server-experimentation",
-    group: "Experimentation",
+    group: "Chaos Experimentation",
```

Comment:
should we add chaos word to `displayName` and `description` fields a few lines below?